### PR TITLE
(major) upgrade to Polymer 3 tools

### PR DESCRIPTION
**Pull request by cristinecula**
* removed bower
* upgraded test runner
* upgraded linter
* fixed demos
* export es modules (#13)

Breaking changes:

* dist: removed import*.html - HTML imports are deprecated
* build: modified IIFE wrapper script to use `window` instead of `this` (top-level `this` is undefined in js modules)

### CODE REVIEW

**Comment by cristinecula** [comment]
@nomego @megheaiulian

File: build.js [hunk 1]
```diff
@@ -42,11 +42,9 @@ new ClosureCompiler([
 	'--warning_level=VERBOSE',
 	'--jscomp_warning=lintChecks',
 	'--compilation_level=ADVANCED',
-	'--isolation_mode=IIFE',
-	'--js=src/trace.js',
-	'--js=src/nulldownloader.js',
-	'--js=src/nullzip.js',
-	'--js=src/nullxlsx.js',
+	'--module_resolution=NODE',
+	'--output_wrapper=(function(){%output%}).call(window);',
```
Comment (megheaiulian) [inline build.js]: Do we need the wrapper for ES module ?

File: build.js [hunk 2]
```diff
@@ -42,11 +42,9 @@ new ClosureCompiler([
...
+	'--js=src/**.js',
 	'--js=exports/nullzip-exports.js',
 	'--js=exports/nullxlsx-exports.js',
```
Comment (megheaiulian) [inline build.js]: Will we still need the exports  ?

File: build.js (hunk 1 above)
Comment (cristinecula) [reply build.js]: There are two ways to import this package:
* `<script src="/path/to/dist/nullxlsx.js">`
This is the script compiled and minified with closure-compiler which has the wrapper. When evaluated, it registers NullXlsx to window.

* `import {NullXlsx} from '@neovici/nullxlsx'`
This imports the raw ES source code. 

So, we need the wrapper to output the minified, register-to-window build.

File: build.js (hunk 2 above)
Comment (cristinecula) [reply build.js]: It's for the minfied, register-to-window build

**Comment by nomego** [comment]
We don't need the minified version so we should drop it to avoid the maintenance.
Can be done outside of this PR though.

**Comment by cristinecula** [comment]
@Wurper @nomego ready for release!

**Pull request merged**